sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormField.cs

```diff
@@ -104,6 +104,21 @@ internal FormField(string name, FieldData labelData, FieldData valueData, FieldV
         /// </summary>
         public float Confidence { get; }
 
+        /// <inheritdoc/>
+        public override string ToString()
+        {
+            // labeled
```

Comment:
As an example of logic into it.
This is not part of guideline. Should be up to the dev